client/payment-methods/index.js

```diff
@@ -341,6 +365,13 @@ const PaymentMethods = () => {
 						) }
 					</PaymentMethodsList>
 				</CardBody>
+			</Card>
+			<br />
+			<Card
+				className={ classNames( 'payment-methods', {
+					'is-loading': status === 'pending',
+				} ) }
```

Comment:
I kept the `className` details same as in https://github.com/Automattic/woocommerce-payments/pull/7210/files#diff-cce9c1ee767cec1c20dc9ee0ce12f48dba6e2b2f9aaf816bf06eec75ca076b96R249-R251 to avoid potential negative impact of this change on the other screens (like e.g. connect account page, [where this component is being used](https://github.com/Automattic/woocommerce-payments/blob/develop/client/connect-account-page/index.tsx#L182))